nuxeo-java-client/src/main/java/org/nuxeo/client/api/ConstantsV1.java

```diff
@@ -118,4 +120,18 @@
 
     public static final String DEFAULT_FILE_CONTENT = "file:content";
 
+    /**
+     * @since 2.4
+     */
+    public static final String USER_FIRST_NAME = "firstName";
+
+    public static final String USER_LAST_NAME = "lastName";
```

Comment:
All the new constants should have a `/** @since 2.4 */`. Also please explain where they can be used.